src/Synth/LFO.cpp

```diff
@@ -289,7 +289,7 @@ float LFO::lfoout()
         computeNextFreqRnd();
     }
             
-    float watch_data[2] = {phase, out};
+    float watch_data[2] = {phaseWithStartphase, out};
```

Comment:
Also, the above `if(phase >= 1)` will behave different now, because the start phase is not included? So this will not catch if `phaseWithStartphase >= 1`. It will also call `computeNextFreqRnd` later now.

Reply:
if(phase >= 1) is only used to calculate a new random amplitude every cycle.
the exact phase where this happens doesn't matter so much because the value is lerped. The effort to put it to the time of the actual "phase reset" is quite high due to possible negative values of startphase.
long story short: i would keep it this way.

Reply:
OK, just to be sure, you have considered that `phase` could now be `>1` in `const float tmp = (incrnd * (1.0f - phase) + nextincrnd * phase);`? Is this acceptable?

Reply:
phase cant be > 1 in that line, because it is limited in line 285 in the tick before

Reply:
Oh, right! resolving this comment.